packages/components/src/components/decorations/arch-decoration-gradient-defs.svg

```diff
@@ -0,0 +1,26 @@
+<svg width="432" height="432" viewBox="0 0 432 432" class="absolute -z-10">
```

Comment:
it was supposed to be just transparent for this svg?

Reply:
This one holds just definitions of gradients so we don't repeat them multiple times in the DOM.